core/components/css-utilities/Display/Display.story.tsx

```diff
@@ -1,68 +1,62 @@
 import * as React from 'react';
-import { Heading, Paragraph, Card, Table, Text } from '@/index';
+import { Heading, Card, Table, Text } from '@/index';
 import utilitiesSchema from '../Schema';
 
 export const display = () => {
   const data = [
     {
       className: 'd-none',
-      properties: 'display: none ;',
+      properties: 'display: none;',
     },
     {
       className: 'd-inline',
-      properties: 'display: inline ;',
+      properties: 'display: inline;',
     },
     {
       className: 'd-inline-block',
-      properties: 'display: inline-block ;',
+      properties: 'display: inline-block;',
     },
     {
       className: 'd-block',
-      properties: 'display: block ;',
+      properties: 'display: block;',
     },
     {
       className: 'd-table',
-      properties: 'display: table ;',
+      properties: 'display: table;',
     },
     {
       className: 'd-table-row',
-      properties: 'display: table-row ;',
+      properties: 'display: table-row;',
     },
     {
       className: 'd-table-cell',
-      properties: 'display: table-cell ;',
+      properties: 'display: table-cell;',
     },
     {
       className: 'd-flex',
-      properties: 'display: flex ;',
+      properties: 'display: flex;',
     },
     {
       className: 'd-inline-flex',
-      properties: 'display: inline-flex ;',
+      properties: 'display: inline-flex;',
+    },
+    {
+      className: 'd-grid',
+      properties: 'display: grid;',
+    },
+    {
+      className: 'd-inline-grid',
+      properties: 'display: inline-grid;',
     },
   ];
   return (
-    <div>
-      <Heading size="xxl">Display property</Heading>
+    <div className="p-4">
+      <Heading size="xxl">Display Utilities</Heading>
       <br />
-      <Text weight="strong">
+      <Text appearance="default">
```

Comment:
Passing `appearance="default"` to the Text component is unnecessary, as it is the default value. Also add weight="strong" to Text.